README.md

```diff
@@ -16,12 +18,16 @@ ember install ember-islands
 
 ## Usage
 
+Add a placeholder for an Ember Component inside your HTML.
```

Comment:
`your existing HTML` or `your existing application HTML` might make it clearer where this goes